module/actor/actor-sheet.js

```diff
@@ -739,5 +806,100 @@ export async function setCovenant(selector, actor) {
         actorUpdate["data.covenant.value"] = found[0].value;
     }
 
+    await actor.update(actorUpdate);
+}
+
+export async function setWounds(selector, actor) {
```

Comment:
Great, I wanted to add such a function.

However, I would have prefered to factorize the damage part from the html part:
Extract all the needed info from the JQuery then call a function of Actor to do the damage itself

Reply:
Mmm, not sure where I could put it. It seems actor-pc is an extension of actor, but I don't know if actor-pc works too for the npcs. Is that what you are talking about?

Reply:
I understand your confusion, it was the same for me. It seems Foundry VTT doesn't  like to have more than one class inheriting from Actor or Item. The original dev tried to do it wihtout success. The name actor-pc is a remain from this period (I deleted npc, covenant, lab since then) 
Of course, there is a trick/hack to do it, I restrained myself until now because it's a hack, but the code is looking more and more like a spagetti dish, so I may do it sooner than later. (If you are interested: https://foundryvtt.wiki/en/development/guides/polymorphism-actors-items)

To answer your question, I was speaking of the ArM5ePCActor class (actor-pc.js). I see a function to setWounds would sit well next to the one removing fatigueLevels.

Reply:
Ok, I added multiple changes, now I will check what you are telling me.

Reply:
So where do you want me to add this? actor-sheet? I did that in there because I was using the same strategy that pickCovenant